main.py

```diff
@@ -1,9 +1,10 @@
 from vkbottle import Bot
+
 from config import api
 from polyschedule.handlers import labelers
 
 
-if __name__ == "__main__":
+if __name__ == '__main__':
     bot = Bot(api=api)
```

Comment:
If `api` is the first parameter, the parameter name is redundant and can be omitted:
```suggestion
    bot = Bot(api)
```